apps/src/blockly/addons/blockSvgLimitIndicator.ts

```diff
@@ -0,0 +1,131 @@
+import {BlockSvg} from 'blockly';
+
+/**
+ * Represents a bubble on a Blockly block, displaying the count of blocks remaining
+ * based on a limit initially stated in the toolbox XML.
+ */
+export default class BlockSvgLimitIndicator {
+  private readonly blockSvg: BlockSvg;
+  private count: number;
+  private readonly bubbleSize: number = 18;
+  private readonly halfBubbleSize: number;
```

Comment:
why don't we set `halfBubbleSize` to 9 here since it's always `bubbleSize / 2`? Also should those `bubbleSize` and `halfBubbleSize` be constants rather than fields since they should never change?

Reply:
Ha, whoops, I was assuming making something private and readonly was perhaps better than a constant. Mike had it that way before. Sorry for giving bad feedback, Mike 🙃

Reply:
No worries, I think a classic constant is more readable here. If I see `this.` I assume it's a changeable field. But I could not be following correct typescript protocol :)

Reply:
Reverted!